playground/package.json

```diff
@@ -6,14 +6,15 @@
   "dependencies": {
     "@codemirror/theme-one-dark": "^6.1.0",
     "@headlessui/react": "^1.7.7",
-    "@tailwindcss/typography": "^0.5.9",
     "@tailwindcss/forms": "^0.5.3",
+    "@tailwindcss/typography": "^0.5.9",
```

Comment:
nit: does this line need to move?

Reply:
What do you mean?

Reply:
this dep was already set, but a line above. So the change was just moving the dependency one line down. This is not a big deal, but it makes it harder to navigate in the code's history.

Reply:
Got it, good point. I was copying a lot of dependencies for this PR so I forgot about that.